neuraxle/metaopt/random.py

```diff
@@ -615,7 +652,11 @@ def fit(self, data_inputs, expected_outputs=None) -> 'BaseStep':
                 started = True
                 self.score = score
                 self.best_validation_wrapper_of_model = copy.copy(step)
+
+                print('\nbest_score: {}'.format(score))
                 best_hyperparams = new_hyperparams
+                print('best_hyperparams: ')
+                print(json.dumps(best_hyperparams.to_nested_dict(), sort_keys=True, indent=4))
```

Comment:
Okay, this is temporary. Will need an hyperparam repo soon. Okay to merge that print, will need to remove later upon having an hparamrepo. 

This makes me think that an hyperparameter respository could be like a `HyperparameterInMemoryPrintRepository`.

Reply:
done in another pr